portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,445 @@
+                  </a>
+                </figure>
+
+                <!-- Fall Fact -->
+                <figure class="effect-goliath tm-gallery-item">
+                  <img src="img/fallFact.jpeg" alt="I love Fall" class="" />
+                  <figcaption>
+                    <h2>
+                      My Favorite Season is
+                      <span>Fall</span>
+                    </h2>
+                    <p>Cozy sweaters + Pumpkin Spice Latte</p>
+                  </figcaption>
+                </figure>
+
+                <!-- Norman Rockwell -->
+                <figure class="tm-gallery-item">
+                  <a href="https://open.spotify.com/album/5XpEKORZ4y6OrCZSKsi46A?si=mUlKVOL3TQSQcvlek9nJvw"
+                    target="_blank">
+                    <img src="img/norman.jpg" alt="Norman Rockwell Album" class="" />
+                  </a>
+                </figure>
+
+                <!-- Nephews Fact -->
+                <figure class="effect-goliath tm-gallery-item">
+                  <img src="img/nephewFact.jpeg" alt="I have 3 nephews" class="" />
+                  <figcaption>
+                    <h2>
+                      I Have Three
+                      <span>Nephews</span>
+                    </h2>
+                    <p>But I see them more as my brothers</p>
+                  </figcaption>
+                </figure>
+
+              </div>
+            </section>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Footer -->
+    <div class="social-footer">
+      <div class="social-wrapper">
+        <ul>
+          <!--Facebook-->
+          <li>
+            <a href="https://www.facebook.com/alex.va99/" target="_blank">
+              <img src="img/facebook.png" alt="My Facebook" class="facebook">
+            </a>
+          </li>
+
+          <!--Instagram-->
+          <li>
+            <a href="https://www.instagram.com/alex.vasqxz/" target="_blank">
+              <img src="img/instagram.png" alt="My Instagram" class="instagram">
+            </a>
+          </li>
+
+          <!--Twitter-->
+          <li>
+            <a href="https://twitter.com/Gustale_xx" target="_blank">
+              <img src="img/twitter.png" alt="My Twitter" class="twitter">
+            </a>
+          </li>
+
+          <!--Linkedln-->
+          <li>
+            <a href="https://www.linkedin.com/in/gustavo-vasquez99/" target="_blank">
+              <img src="img/linkedln.png" alt="My Linkedln" class="linkedin">
+            </a>
+          </li>
+
+          <!--Github-->
+          <li>
+            <a href="https://github.com/alexvasqxz" target="_blank">
+              <img src="img/github.png" alt="My Github" class="github">
+            </a>
+          </li>
+        </ul>
+      </div>
+      <nav class="footer-nav" role="navigation">
+        <p>|| Copyright &copy; 2021 Gustavo Vásquez || All rights reserved || </p>
+      </nav>
+    </div>
```

Comment:
Optional: similar comment as the first one, these blocks seem to be common among all of your pages